gen/test-html.hs

```diff
@@ -195,6 +219,7 @@ meetups = checkMeetupsIndex
             ]
         , time = Just $ read "2016-06-28 18:30:00 +02:00"
         , participants = Just 8
+        , sponsors = []
```

Comment:
According to my calendar/lifelog, this one was at iXperta offices and could maybe be considered as sponsored by iXperta.